packages/core/admin/admin/src/pages/MarketplacePage/components/SortFilter/index.js

```diff
@@ -1,28 +1,47 @@
 import React from 'react';
-import { SimpleMenu, MenuItem } from '@strapi/design-system/SimpleMenu';
+import { Select, Option } from '@strapi/design-system/Select';
 import { useIntl } from 'react-intl';
-import { NavLink } from 'react-router-dom';
+import { Typography } from '@strapi/design-system/Typography';
+import PropTypes from 'prop-types';
 import { getTrad } from '../../../../content-manager/utils';
 
-const SortFilter = () => {
+const SortFilter = ({ value, onChange }) => {
   const { formatMessage } = useIntl();
 
   return (
-    <SimpleMenu
-      variant="tertiary"
-      label={formatMessage({
-        id: getTrad('sort.label'),
-        defaultMessage: 'Sort by',
-      })}
+    <Select
+      id="marketplaceSortButton"
+      size="S"
+      value={value}
+      onChange={(sort) => onChange(sort)}
+      customizeContent={() => (
+        <Typography variant="pi" fontWeight="bold">
+          {formatMessage({
+            id: getTrad('admin.pages.MarketPlacePage.sort.sortBy'),
```

Comment:
You can remove getTrad and use the string directly (same for the other translations here)